DFKPS/DFKPS-IOC-01App/Db/DFKPS_8000_status.db

```diff
@@ -77,3 +77,45 @@ record(ao, "$(P)SIM:CURR"){
     field(EGU, "A")
 }
 alias("$(P)SIM:CURR", "$(P)SIM:SETIRAW")
+
+record(ao, "$(P)SIM:RAW"){
+field(DTYP, "Soft Channel")
+field(EGU, "A")
+}
+alias("$(P)SIM:CURR", "$(P)SIM:RAW:SP")
+alias("$(P)SIM:CURR", "$(P)SIM:RAW")
+
+record(ai, "$(P)RAW")
```

Comment:
Would it be worth making this `$(P)CURR:RAW`? I am undecided, what do you think?